### PR TITLE
chore: update unify config doc

### PR DESCRIPTION
**Pull request by xiaolang124**
1. Add collaborate cautions and benefits to doc.
2. Add help link to upgrade question.

### CODE REVIEW

File: packages/fx-core/src/core/middleware/consolidateLocalRemote.ts [hunk 1]
```diff
@@ -38,6 +38,8 @@ import * as util from "util";
 import { ManifestTemplate } from "../../plugins/resource/spfx/utils/constants";
 
 const upgradeButton = "Upgrade";
+const LearnMore = "Learn More";
+const LearnMoreLink = "https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-to-support-unify-config";
```
Comment (kimizhu) [inline packages/fx-core/src/core/middleware/consolidateLocalRemote.ts:42]: Use aka link instead.

File: packages/fx-core/resource/unify-config-change-logs.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 We have updated the configuration files so that your project is compatible with the latest Teams Toolkit features, including a consistent configuration file schema and a single Teams app manifest template across local and remote environments.
 
+> Important Notes: If you collaborate on this project with your co-workers, please ensure your team members update the Teams Toolkit extension to the latest version after committing the changes with this upgrade.
```
Comment (kimizhu) [inline packages/fx-core/resource/unify-config-change-logs.md:5]: @MuyangAmigo can you help to review this?

File: packages/fx-core/src/core/middleware/consolidateLocalRemote.ts (hunk 1 above)
Comment (xiaolang124) [reply packages/fx-core/src/core/middleware/consolidateLocalRemote.ts]: Already changed to ```https://aka.ms/teamsfx-unify-config-guide```.

File: packages/fx-core/resource/unify-config-change-logs.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/fx-core/resource/unify-config-change-logs.md:5]: This is what I have updated. Do you have any particular comments?

Comment (kimizhu) [reply packages/fx-core/resource/unify-config-change-logs.md:5]: I saw it was different from what in the wiki page, so was wonder which one we should use. 
Now I see wiki page is also updated with this change, so no more question here. Thanks @MuyangAmigo and @xiaolang124

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**